server/src/postgres/init.rs

```diff
@@ -27,6 +27,7 @@ pub async fn create_db() -> Result<(), ArrErr> {
     ResourceObject::<itinerary::Data>::init_table().await?;
     ResourceObject::<itinerary_flight_plan::Data>::init_table().await?;
     ResourceObject::<parcel::Data>::init_table().await?;
+    ResourceObject::<parcel_scan::Data>::init_table().await?;
```

Comment:
Should this be init'd after the `scanner` table since there is a foreign key constraint?

Reply:
[cc969e4](https://github.com/Arrow-air/svc-storage/pull/62/commits/cc969e45bda8b46c79ae04c6c37063bdc700886e)